src/Tribe/Views/V2/View.php

```diff
@@ -2194,6 +2194,12 @@ protected function should_show_recent_past_events_view() {
 	 */
 	protected function setup_additional_views( array $events = [], array $template_vars = [] ) {
 
+		// Only return if on the current or upcoming view and not in the past.
+		// todo this still displays on day view in the past if it is
+		if ( $template_vars[ 'is_past' ] ) {
```

Comment:
:no_entry_sign: Array keys must NOT be surrounded by spaces if they only contain a string or an integer.

<sub>hash: 7c977e2483399188853f441cc61747c9</sub>